api/__init__.py

```diff
@@ -1,3 +1,2 @@
-from flask import Flask 
```

Comment:
Why remove it? it is required for making the app var

Reply:
yeah sry accidental